src/compiler/scala/tools/nsc/reporters/StoreReporter.scala

```diff
@@ -21,6 +21,8 @@ import scala.reflect.internal.util.Position
 class StoreReporter(val settings: Settings) extends FilteringReporter {
   import StoreReporter._
 
+  def this() = this(new Settings)
```

Comment:
So here's the thing, usually I just don't care. With a plain Reporter, I can let the filter-forward thing just do it, it's essentially the compiler's responsibility, I just get the info it chooses to emit (as per normal).